_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -0,0 +1,438 @@
+
+```html
+<html lang="en">
+	<p>
+		We are on social media:
+		<a href="http://facebook.com">Follow us</a>
+		<a href="http://twitter.com">Follow us</a>
+	</p>
+</html>
+```
+
+#### Failed Example 2
+
+The same [accessible name][] is used for two links in the same [context][programmatically determined link context] going to web pages that are similar, but have different information in their content.
+
+```html
+<html lang="en">
+	<p>
+		Learn more (<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html"
+			>Contact us</a
+		>) and get in touch (
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html"
+			>Contact us</a
+		>)
+	</p>
+</html>
+```
+
+#### Failed Example 3
+
+These two HTML `span` elements have an [explicit role][] of link, but lead to resources that offer different content.
+
+```html
+<html lang="en">
+	<p>
+		<span
+			role="link"
+			tabindex="0"
+			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html'"
+		>
+			My university
+		</span>
+
+		<span
+			role="link"
+			tabindex="0"
+			onclick="location='/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html'"
+		>
+			My university
+		</span>
+	</p>
+</html>
+```
+
+#### Failed Example 4
+
+The same [accessible name][] is used for image links in the same [context][programmatically determined link context] going to different resources.
+
+```html
+<html lang="en">
+	<p>
+		We are on social media:
+		<a href="http://facebook.com"><img src="facebook.jpg" alt="Follow us"/></a>
+		<a href="http://twitter.com"><img src="twitter.jpg" alt="Follow us"/></a>
+	</p>
+</html>
+```
+
+#### Failed Example 5
+
+A set of two SVG `a` elements that have the same [accessible name][] and [context][programmatically determined link context] but link to different resources.
+
+```html
+<html lang="en">
+	<p>
+		<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+			<a href="http://facebook.com" aria-label="Follow us">
+				<circle cx="50" cy="40" r="35" />
+			</a>
+
+			<a href="http://twitter.com">
+				<text x="50" y="90" text-anchor="middle">
+					Follow us
+				</text>
+			</a>
+		</svg>
+	</p>
+</html>
+```
+
+#### Failed Example 6
+
+Both links resolve to [same resource][] after redirect, but the redirect is not instant.
+
+```html
+<html lang="en">
+	<p>
+		Learn more (<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/index.html"
+			>Contact us</a
+		>) and get in touch (<a
+			href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/redirect1.html"
+			>Contact us</a
+		>)
+	</p>
+</html>
+```
+
+### Inapplicable
+
+#### Inapplicable Example 1
+
+These `a` and `area` elements do not have a role of `link`.
+
+```html
+<html lang="en">
+	<a>Link text</a>
+	<area aria-label="Link text" />
```

Comment:
That's two test cases in one. Can you split them out please?

Reply:
Not sure I understand what you mean. What is being tested here besides elements that do not have the link role?